src/commands/claim/index/processor.test.ts

```diff
@@ -144,9 +144,11 @@ describe("claim", () => {
         msgOrInteraction: msg,
         title: "Failed to claim tip!",
         description: `Mochi wallet's balance is insufficient to proceed this transaction.\n${getEmoji(
-          "POINTINGRIGHT"
+          "ANIMATED_POINTING_RIGHT",
+          true
         )} You can contact our developer at suggestion forum in [Mochi Discord](${DISCORD_URL})!\n${getEmoji(
-          "POINTINGRIGHT"
+          "ANIMATED_POINTING_RIGHT",
+          true
         )} You can try to claim other tips and get back to this one later! ${getEmoji(
           "soon"
         )}\nSorry for this inconvenience ${getEmoji("nekosad")}`,
```

Comment:
nekosad should be uppercase

Reply:
k nằm trong changes của PR này nhe, cái này a sẽ sửa ở PR sau